framework/src/play/server/JakartaServletWrapper.java

```diff
@@ -555,8 +563,27 @@ public void run() {
 
         @Override
         public void execute() throws Exception {
+            response.onWriteChunk(chunk -> writeChunk(chunk));
             ActionInvoker.invoke(request, response);
             copyResponse(request, response, httpServletRequest, httpServletResponse);
+
+        }
+
+        private void writeChunk(Object chunk) {
+            try {
+                    OutputStream out = httpServletResponse.getOutputStream();
+                    byte[] bytes;
+                    if (chunk instanceof byte[]) {
+                        bytes = (byte[]) chunk;
+                    } else {
+                        String message = chunk == null ? "" : chunk.toString();
+                        bytes = message.getBytes(response.encoding);
```

Comment:
The wiki says you should write the length length in hex followed by `\r\n`, then the content, then another `\r\n` and the end should be a `0\r\n\r\n`. It could be that Facebook "knows" how to handle it... but it could lead to tricky behaviours, don't you think?
https://en.wikipedia.org/wiki/Chunked_transfer_encoding